s3storageprovider/src/main/java/org/duracloud/s3storage/S3StorageProvider.java

```diff
@@ -70,6 +75,11 @@
     private static final StorageClass DEFAULT_STORAGE_CLASS =
         StorageClass.Standard;
 
+    private static final String UTF_8 = StandardCharsets.UTF_8.name();
+    
+    private static final String HEADER_VALUE_PREFIX = UTF_8 + "' '";
```

Comment:
The example given in https://tools.ietf.org/html/rfc5987#section-3.2.2 leaves no space between single quote characters when the optional language value is omitted. I'd recommend removing the space between the quotes.